GenericApp/app/src/main/java/io/openremote/app/util/UrlUtils.kt

```diff
@@ -0,0 +1,20 @@
+package io.openremote.app.util
+
+object UrlUtils {
+    fun isIpAddress(url: String): Boolean {
+        val ipPattern = Regex(
+            "^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(:\\d{1,5})?$"
+        )
+        return ipPattern.matches(url)
+    }
+    fun startsWithHttp(url: String): Boolean {
+        return url.startsWith("http://") || url.startsWith("https://")
+    }
+
+    fun endsWithTld(url: String): Boolean {
+        val tldPattern = Regex(
+            ".*\\.[a-z]{2,6}$"
```

Comment:
TLDs are now more complex than this, for instance check list at [Root Zone Database](https://www.iana.org/domains/root/db)
Maybe use something like this (not tested): https://regex101.com/library/eV8jM7?filterFlavors=pcre&orderBy=MOST_POINTS&page=7&search=url